### PR TITLE
Probabilities calculation for decision forest has been added

### PR DESCRIPTION
**Pull request by masdevas**

### CODE REVIEW

**Comment by masdevas** [comment]
https://ecmd.inn.intel.com/commander/link/jobDetails/jobs/e9fa89da-dada-f1ae-8eeb-8cdcd4b723a1

**Comment by masdevas** [comment]
http://intel-ci.com/commander/link/jobDetails/jobs/e9fa89da-dada-f1ae-8eeb-8cdcd4b723a1

**Comment by masdevas** [comment]
http://intel-ci.intel.com/e9fa89da-dada-f1ae-8eeb-8cdcd4b723a1

**Comment by ShvetsKS** [comment]
make test

**Comment by ShvetsKS** [comment]
make test

**Comment by ShvetsKS** [comment]
make test

File: algorithms/kernel/dtrees/dtrees_model_impl.h [hunk 1]
```diff
@@ -466,6 +491,7 @@ class ModelImpl
         {
             arch->setSharedPtrObj(_impurityTables);
             arch->setSharedPtrObj(_nNodeSampleTables);
+            arch->setSharedPtrObj(_probTbl);
```
Comment (ShvetsKS) [inline algorithms/kernel/dtrees/dtrees_model_impl.h]: move it under separate if branch
daalVersion >= COMPUTE_DAAL_VERSION(2020, 0, 0)

**Comment by masdevas** [comment]
make test

**Comment by masdevas** [comment]
make test

**Comment by masdevas** [comment]
make test

**Pull request merged**